exercises/dnd-character/src/test/java/DnDCharacterTest.java

```diff
@@ -102,24 +102,29 @@ public void testAbilityModifierForScore18Is4() {
         assertEquals(4, dndCharacter.modifier(18));
     }
 
+    private boolean inRange(int ability) {
+        return ability >= 3 && ability <= 18;
+    }
+
     @Ignore("Remove to run test")
     @Test
     public void testRandomAbilityIsWithinRange() {
-        int score = dndCharacter.ability();
-        assertTrue(score > 2 && score < 19);
+        for (int i = 0; i < 1000; i++) {
+            assertTrue(inRange(dndCharacter.ability()));
+        }
     }
 
     @Ignore("Remove to run test")
     @Test
     public void testRandomCharacterIsValid() {
-        for (int i = 0; i < 1000; i++) {
+        for (int i = 0; i < 100; i++) {
```

Comment:
100 should really be enough, as with 1000 it is the most expensive test. This is essentially the same as `testRandomAbilityIsWithinRange`.